csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -82,6 +82,7 @@
   </Target>
 
   <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.6.1" />
```

Comment:
remove this dependency from src, and make it a dependency of the test project

Reply:
Same for the OnnxML.cs

Reply:
done.